.travis.yml

```diff
@@ -8,7 +8,6 @@ install:
  - export DEBIAN_FRONTEND=noninteractive
  - sudo apt-get update
  - sudo apt-get -y install
-        chromium-browser
```

Comment:
Heh, chromium-browser is already installed on Travis so it runs the virtual-stb tests anyway.